Docs/source/pages/get-started.md

```diff
@@ -273,27 +273,43 @@ AltTester package contains AltDriver class used to connect to the instrumented a
 
     .. tab:: C#
```

Comment:
You should also change to C#-Unity because it will need a running AltServer

Reply:
resolved